README.md

```diff
@@ -172,27 +188,86 @@ command.
 There are three views for reviewing a pull request on GitHub: **Conversation**,
 **Commits**, and **Files changed**.
 
-#### Conversation ####
+**Conversation**
+
+- Comments on the pull request show up here, including any inline comments that
+  are added when viewing the file diffs. A summary of submitted commits is also
+  listed.
+- Use this section to submit general feedback about the pull request.
+
+**Commits**
+
+- A more detailed list of commits included in the pull request, that will allow
+  a reviewer to view each individually.
+- Usually before merging, you'll want all the commits from that change to be
+  squashed into a single commit. This so that the commit can show the whole
+  change that was made, and so that there aren't a bunch of typo fixes in the
```

Comment:
How about this?

"Some of the reasons for this is so that the commit can show the whole
change that was made, and so that there aren't a bunch of typo fixes in the..."